su-template/default/module/html/method_summary.erb

```diff
@@ -0,0 +1,16 @@
+<% if method_listing.size > 0 %>
+  <% groups(method_listing) do |list, name| %>
+    <h2>
+      <a name="<%= name.gsub(' ', '_').downcase %>">
```

Comment:
Looking a little close, I'm tempted at doing this:

```
    <h2>
      <%= name %>
      <a id="<%= name.gsub(' ', '_').downcase %>" href="#<%= name.gsub(' ', '_').downcase %>">#</a>
      <small><a href="#" class="summary_toggle">collapse</a></small>
    </h2>
```

That gives us a visual anchor that we can click on to align the page to that header - also a convenient way to right-click and copy the URL.

![2016-12-23_12h33_06](https://cloud.githubusercontent.com/assets/192418/21453064/fa79eb2e-c90b-11e6-8c66-07cbdeef5271.png)